JustTrack/Classes/JustTrack/Tracking.swift

```diff
@@ -95,23 +95,19 @@ public class EventTracking: NSObject {
     /// - seealso: `TrackerConsole`.
     @discardableResult
     public func loadDefaultTracker(_ type: TrackerType) -> Bool {
```

Comment:
If this method always succeeds, maybe worth changing the return type to void?

Reply:
@dchakarov I thought about it but that would cause a major bump, which is what I don't want now. @akuzminskyi is also working on more changes #32, #33, #35, #36 and more fundamental to come I'm sure, that we can pack all together in a major release.